sotodlib/coords/helpers.py

```diff
@@ -374,6 +374,10 @@ def get_focal_plane_cover(tod=None, count=0, focal_plane=None,
         eta = focal_plane.eta
     else:
         xi, eta = xieta[:2]
+    nans = ~np.isnan([xi, eta])
```

Comment:
This can't be added like this. You're cutting up an array most likely to be in an axis manager without running through AxisManager axes. 


Lots of other discussion related to this issue: https://github.com/simonsobs/sotodlib/issues/763